tests/subsys/emds/emds_flash/src/main.c

```diff
@@ -658,18 +662,26 @@ ZTEST(emds_flash_tests, test_write_speed)
 	dk_set_led(0, false);
 	toc = k_uptime_ticks();
 	store_time_us = k_ticks_to_us_ceil64(toc - tic);
-	printk("Storing 4 bytes took: %lldus\n", store_time_us);
-	zassert_true(store_time_us < 300, "Storing 4 bytes took to long time");
+	printk("Storing %d bytes took: %lldus\n", sizeof(data_in), store_time_us);
+#if defined CONFIG_SOC_FLASH_NRF_RRAM
+	zassert_true(store_time_us < 400, "Storing %d bytes took to long time", sizeof(data_in));
+#else
+	zassert_true(store_time_us < 200, "Storing %d bytes took to long time", sizeof(data_in));
+#endif
 
 	dk_set_led(0, true);
 	tic = k_uptime_ticks();
 	emds_flash_write(&ctx, 2, data_in_big, sizeof(data_in_big));
 	dk_set_led(0, false);
 	toc = k_uptime_ticks();
 	store_time_us = k_ticks_to_us_ceil64(toc - tic);
-	printk("Storing 1024 bytes took: %lldus\n", store_time_us);
-#if !defined CONFIG_SOC_FLASH_NRF_RRAM /* TODO: Fix it with NCSDK-26922 */
-	zassert_true(store_time_us < 13000, "Storing 1024 bytes took to long time");
+	printk("Storing %d bytes took: %lldus\n", sizeof(data_in_big), store_time_us);
+#if defined CONFIG_SOC_FLASH_NRF_RRAM
+	zassert_true(store_time_us < 6848, "Storing %d bytes took to long time",
```

Comment:
There are quite a few somewhat magic numbers here, could you create\
`#define EXPECTED_STORE_TIME_RRAM_1024 (6848)` etc. for this and all other values used to compare `store_time_us` to make this a bit easier to read for people who aren't familiar with the test?